utils/run_helpers.py

```diff
@@ -109,4 +109,11 @@ def append(block, parent_path: str, id_with_prefix: str):
                         id_with_prefix=f"{id_with_prefix}-{i}",
                     )
 
-    append(entire_pipeline, Const.output_pipelines_path, "")
+    append(entire_pipeline, pipeline.experiment.project_name, "")
+
+
+def add_experiment_config_to_blocks_(
+    pipeline: Pipeline, experiment: Experiment
+) -> None:
+    for model in flatten(pipeline.children()):
```

Comment:
ouch, is this necessary? unfortunately Experiment has everything on it, you can start mutating it within a Model! that's probably not great. is there any alternative, or can we just pass in the necessary information (that we copy, so a Model won't be able to modify the pipeline?)

Reply:
or am I misunderstanding something?

Reply:
There were two attributes that we wanted to add: 
- ``project_id``
- ``train:bool``

I think its ok that we add it to the models, but we should copy it, I agree! Changing it now.

Reply:
I fixed this by copying the experiment attributes!